### PR TITLE
Tiddit for tumor varcalling

### PR DESCRIPTION
**Pull request by WackerO**
This PR adds tiddit for tumor variant calling analogous to https://github.com/nf-core/sarek/pull/581.

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/local/pair_variant_calling.nf [hunk 1]
```diff
@@ -41,6 +43,7 @@ workflow PAIR_VARIANT_CALLING {
     strelka_vcf          = Channel.empty()
     msisensorpro_output  = Channel.empty()
     mutect2_vcf          = Channel.empty()
+    tiddit_vcf          = Channel.empty()
```
Comment (maxulysse) [inline subworkflows/local/pair_variant_calling.nf:46]:
```suggestion
    tiddit_vcf           = Channel.empty()
```

**Comment by WackerO** [comment]
Just removed pair VC, will look at that separately so that tumor tiddit can be merged already. Note: I checked the output vcf, both tumoronly and germline produce a vcf that does not contain any variants, is that supposed to be the case? The vcf ends like this:

##TIDDITcmd="/usr/local/bin/tiddit --sv --skip_assembly --bam test.paired_end.recalibrated.sorted.cram --ref genome.fasta -o sample1" #CHROM  POS     ID      REF     ALT     QUAL    FILTER  INFO    FORMAT  normal

**Comment by WackerO** [comment]
Oh boy, I'll update the subway map and hope that Susi does not find any 0.1 nm wide bumps to complain about :*D
Sounds good!

File: subworkflows/local/tumor_variant_calling.nf [hunk 2]
```diff
@@ -29,6 +30,7 @@ workflow TUMOR_ONLY_VARIANT_CALLING {
         panel_of_normals_tbi          // channel: [optional]  panel_of_normals_tbi
         chr_files
         mappability
+        bwa                           // channel: [optional] bwa
```
Comment (maxulysse) [inline subworkflows/local/tumor_variant_calling.nf:33]: Can the reference input be listed alphabetically? (you'll need to reorganize when calling this subworkflow too)

Comment (WackerO) [reply subworkflows/local/tumor_variant_calling.nf:33]: Oh of course! The other two varcalling subworkflows are not alphabetical, either, should I sort those as well?

Comment (FriederikeHanssen) [reply subworkflows/local/tumor_variant_calling.nf:33]: cram should stay at the top though, but all the references can be sorted

**Review by FriederikeHanssen — APPROVED** [review]

**Pull request merged**